### PR TITLE
chore(anchor): update external anchor security docs

### PR DESCRIPTION
**Pull request by austin94**
## Description

Documentation for https://github.com/zendeskgarden/react-components/pull/821

Holding as draft until the next release.

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/pages/components/anchor.mdx [hunk 1]
```diff
@@ -63,6 +63,17 @@ import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExterna
   <AnchorExternal />
 </CodeExample>
 
+#### Security
```
Comment (jzempel) [inline src/pages/components/anchor.mdx:66]: Positioning and hierarchy is probably not ideal. We need to come up with a rule for how/where this kind of content lives. Is it just paragraph text under the component heading (under `###Anchor` below)? Or should this be removed and the `isExternal` prop description amplified under `react-components`? (My opinion is that it makes more sense as a prop description.)

File: src/pages/components/anchor.mdx [hunk 2]
```diff
@@ -63,6 +63,17 @@ import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExterna
...
+
+When linking to third-party content you may be [exposing your site to performance and security issues](https://web.dev/external-anchors-use-rel-noopener/).
+Garden provides two attributes by default for external anchors:
+
+- `target="_blank"`
+- `rel="noopener noreferrer"`
+
+These attributes ensure the third-party content is ran in a unique process
+and unable to access the current pages `window` object.
```
Comment (jzempel) [inline src/pages/components/anchor.mdx:75]: Grammar:

- "...ensure the third-party content is ran in a unique process" vs "...ensure ~the~ third-party content runs in a unique process"
- "...current pages `window`" vs "...current page's `window`"

File: src/pages/components/anchor.mdx [hunk 3]
```diff
@@ -63,6 +63,17 @@ import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExterna
   <AnchorExternal />
 </CodeExample>
 
+#### Security
+
+When linking to third-party content you may be [exposing your site to performance and security issues](https://web.dev/external-anchors-use-rel-noopener/).
```
Comment (jzempel) [inline src/pages/components/anchor.mdx:68]: Seems pretty unfortunate that a description of external links, with an external URL, isn't being properly formatted with an external link treatment. Perhaps this gets sorted when markdown anchors are wrapped Gatsby [`Link`](https://github.com/zendeskgarden/website/blob/962427a3b36640a811501593b9f04e5b701a0dc1/src/layouts/Root/components/StyledNavigationLink.tsx#L20-L36) components.

**Comment by jzempel** [comment]
@austin94 I'd recommend simplifying and moving over to the `isExternal` prop description in `react-components` with the following:

> Used when the anchor navigates to an external resource. Applies `target="_blank"` along with `rel="noopener noreferrer"` to ensure cross-origin destination links are [safe](https://web.dev/external-anchors-use-rel-noopener/).

File: src/pages/components/anchor.mdx (hunk 2 above)
Comment (allisonacs) [reply src/pages/components/anchor.mdx:75]: Also:

> These attributes ensure third-party content runs in a unique process and _can't_ access the current page's window object.

Make sure those `'` get keyed in as proper apostrophes please!

**Comment by allisonacs** [comment]
If @jzempel feels we should move it over to the props, let's do that. I left a minor grammar fix.

**Pull request closed without merging**